packages/rafz/src/index.ts

```diff
@@ -201,7 +212,13 @@ function eachSafely<T>(values: Eachable<T>, each: (value: T) => void) {
 /** Tree-shakable state for testing purposes */
 export const __raf = {
   /** The number of pending tasks */
-  count: 0,
+  count(): number {
```

Comment:
It'd be helpful for me to understand why we've moved `count` to a pseudo `getter` function with a global variable.

Reply:
My understanding from the comments is that `__raf` is intended to be a container for data/functions that are used only in the tests. As of this PR, the `count` variable is now needed for actual control flow rather than just for test observability, so I moved it to a global.